lightning/api_clncommando.go

```diff
@@ -5,19 +5,19 @@ import (
 	"time"
 )
 
-// ClnCommandoAPI struct
-type ClnCommandoAPI struct {
-	ClnSocketRawAPI
+// ClnCommandoLightningAPI struct
+type ClnCommandoLightningAPI struct {
+	ClnRawLightningAPI
 }
 
 // Compile time check for the interface
-var _ LightingAPICalls = &ClnCommandoAPI{}
+var _ LightingAPICalls = &ClnCommandoLightningAPI{}
 
-// NewClnCommandoLightningAPIRaw gets a new API (not that addr is in pubkey@address format)
+// NewClnCommandoLightningAPIRaw gets a new API (note that addr is in pubkey@address format)
 func NewClnCommandoLightningAPIRaw(addr, rune string) LightingAPICalls {
-	api := &ClnCommandoAPI{}
+	api := &ClnCommandoLightningAPI{}
```

Comment:
why not return here

Reply:
Because we initialize some stuff later on, this change was just part of the naming sync.